yb_migrate/src/migration/oracle.go

```diff
@@ -77,6 +77,7 @@ func Ora2PgExtractSchema(source *srcdb.Source, exportDir string) {
 					log.Infof("ERROR in output scanner goroutine: %s", line)
 					runtime.Goexit()
 				}
+				log.Infof("ora2pg STDOUT: %s", outScanner.Text())
```

Comment:
Save the result of the `outScanner.Text()` in a local variable and use it everywhere.

Reply:
I recommend using the `bytes.Buffer` approach (to capture STDOUT/STDERR) here as well.

Reply:
nit: add this line in `else` condition, since if statement already have log

Reply:
Is there a need to save outScanner.Text() in a local variable? It is used sparingly in the code segment

Reply:
if condition logs whenever an error is caught, I have purposely kept this log statement ahead of the error-catching If statement due to the `runtime.Goexit()` statement within the if condition

Reply:
I have made this change for readability in the latest commit 👍🏽